cristin-keyword/src/test/resources/nvaFetchKeywordResponse.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Went for this model for keyword. I also have a task using this model for keyword query and possibly country list